src/c/c.cpp

```diff
@@ -0,0 +1,2835 @@
+/**
+ * Copyright 2022 AntGroup CO., Ltd.
```

Comment:
In the implementation directory, it is recommended to rearrange this to `src/lgraph_api/c.cpp`.